core/src/main/scala/com/nvidia/spark/rapids/tool/analysis/AppSQLPlanAnalyzer.scala

```diff
@@ -200,7 +240,16 @@ class AppSQLPlanAnalyzer(app: AppBase, appIndex: Int) extends AppAnalysisBase(ap
   }
 
   // Store (min, median, max, total) for a given metric
-  private case class statisticsMetrics(min: Long, med:Long, max:Long, total: Long)
+  private case class StatisticsMetrics(min: Long, med:Long, max:Long, total: Long)
+
+  private object StatisticsMetrics {
+    def getOrZerofill(src: Option[StatisticsMetrics]): StatisticsMetrics = {
```

Comment:
The OrZeroFill in the name seems unnessary, it should return either existing or new one that should be initialized to proper values, in this case that would be all 0 so the name seems extra unless you had other uses where it initialized differently.  

This honestly just seems unnecessary, this is just .getOrElse(StatisticsMetrics(0L, 0L, 0L, 0L))
I think that would be more readable then having to jump to a function.

Reply:
You are right.
Removed this method and simplified the code using getOrElse